docs/install.md

```diff
@@ -35,20 +38,22 @@ For more details about MongoDB extension and library for PHP check [here][PHPMon
 [MongoDBMac]: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/
 [MongoDBLinux]: https://docs.mongodb.com/manual/administration/install-on-linux/
 [MongoDBWindows]: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/
+[removedhttp]: https://docs.mongodb.com/manual/release-notes/3.6-compatibility/#http-interface-and-rest-api
 
```

Comment:
It is already available in the mongodb_watchdog.md file under mongodb_watchdog.request_tracking variable

Reply:
Nice changes, and the footnotes extension appears to work well. I just noticed the modules/mongodb/ page also contains references to the old Simpletest namespaces. The explanation about static::$modules doesn't match the code, either, and I'm not sure it is really applicable: look at the watchdog/storage tests for details. The bit about extending settings does not reflect the current state of the base test class, which provides a getSettingsArray method for that purpose.

Reply:
I verified in 2988770-watchdog_test_phpunit branch and also in default 8.x-2.x branch but the static::$modules variable is being used in even watchdog and KV modules. It would be better if you can provide more clarity on that point. I have updated the settings extension part.

Reply:
Just looked at the code again, and the code doesn't add to `static::$modules`, but redefines it entirely in every case as you can see, by just redefining the static property every time. Adding to it as the doc suggests would have to be something like:

```php
class FooTest extends WebTestBase {
  // No static redefined here
  // static $modules = [...whatever...]

  public function __construct($name = NULL, array $data = [], $dataName = '') {
    // But extend the existing list here.
    self::$modules[] = 'foo';
    parent::__construct($name, $data, $dataName);
  }
```

But even then, the static property would only be modified upon the test instance creation, and would risk being modified multiple times if the test is instantiated several times, so I'm not sure it would work if PhpUnit starts using static methods before instantiating the test. IMO it is best to do as the existing tests do, and just redefine the whole list of modules in every test instead of relying on this brittle "extension" mechanism in the constructor. 
So it's mostly a question of formulation, saying the test must define the entire list including the list in the parent tests instead of talking about "adding" instead of "redefining", because what the existing tests do is indeed "redefining". What's your take on this ?

Reply:
Yes. As per current document the code would be like what you have given above. But as you said, there is possibility that if the test is instantiated multiple types then below line would add the module more than 1 time.

```php
self::$modules[] = 'foo';
```
  So there are 2 ways for this. 

1. Follow above method of adding but also have a `__destruct` method that would re-initialise `self::$modules` by removing the current module name or empty it completely. I am not sure if this would work but just an idea.

2. This is the easy and safe method where it is re-defining the `self::$modules` as done in KV and Watchdog tests.

I would also suggest 2nd option. Let me know your opinion then I will modify docs accordingly

Reply:
Second option for me too, let's roll that.